conf/routes

```diff
@@ -4,112 +4,205 @@ GET        /index.html                 @org.sagebionetworks.bridge.play.controll
+POST   /api/v2/surveyresponses/:guid                                @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.appendSurveyAnswersV1(guid: String)
+GET    /api/v2/surveys/:surveyGuid/revisions/published              @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyMostRecentlyPublishedVersionV1(surveyGuid: String)
+GET    /api/v2/surveys/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyController.getSurveyV1(surveyGuid: String, createdOn: String)
+POST   /api/v2/surveys/:surveyGuid/revisions/:createdOn             @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponseV1(surveyGuid: String, createdOn: String)
+POST   /api/v2/surveys/:surveyGuid/revisions/:createdOn/:identifier @org.sagebionetworks.bridge.play.controllers.SurveyResponseController.createSurveyResponseWithIdentifierV1(surveyGuid: String, createdOn: String, identifier: String)
 
 # API - Schedules
-GET    /api/v1/schedules   @org.sagebionetworks.bridge.play.controllers.ScheduleController.getSchedules
+GET    /api/v1/schedules   @org.sagebionetworks.bridge.play.controllers.ScheduleController.getSchedulesV1
 
 # API - Tasks
-GET    /api/v1/tasks       @org.sagebionetworks.bridge.play.controllers.TaskController.getTasks(until: java.lang.String ?= null)
-POST   /api/v1/tasks       @org.sagebionetworks.bridge.play.controllers.TaskController.updateTasks()
+GET    /api/v1/tasks       @org.sagebionetworks.bridge.play.controllers.TaskController.getTasksV1(until: java.lang.String ?= null)
+POST   /api/v1/tasks       @org.sagebionetworks.bridge.play.controllers.TaskController.updateTasksV1()
 
 # API - Consent
 POST   /api/v1/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.giveV1
 POST   /api/v2/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.giveV2
-GET    /api/v1/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.getConsentSignature
-GET    /api/v2/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.getConsentSignature
-POST   /api/v1/consent/email                 @org.sagebionetworks.bridge.play.controllers.ConsentController.emailCopy
-POST   /api/v1/consent/dataSharing/suspend   @org.sagebionetworks.bridge.play.controllers.ConsentController.suspendDataSharing
-POST   /api/v1/consent/dataSharing/resume    @org.sagebionetworks.bridge.play.controllers.ConsentController.resumeDataSharing
-POST   /api/v2/consent/dataSharing           @org.sagebionetworks.bridge.play.controllers.ConsentController.changeSharingScope
+GET    /api/v1/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.getConsentSignatureV1
+GET    /api/v2/consent                       @org.sagebionetworks.bridge.play.controllers.ConsentController.getConsentSignatureV1
```

Comment:
Now I'm really confused. How is it that this worked before, but not the other thing?